main.js

```diff
@@ -0,0 +1,25 @@
+const menuIcon = document.getElementById('menu-id');
```

Comment:
**Kindly work on these requested changes:**
- Fix style issues at the top of your menu once that it's displayed it's missing a few pixels to cover totally the page, kindly add this to your ``background-color``.
- The 'X' for close the menu should be nice to have it to a few pixels to the right.

_Kindly try to follow the style guidelines in yout Figma template._